lib/prompts.ts

```
@@ -0,0 +1,139 @@
+
+/**
+ * System prompt for the module-specific chat assistant.
+ * This is used when the user is chatting within a specific module.
+ *
+ * @param moduleContext - Information about the module and its resources
+ * @param searchContext - Optional search results to include in the prompt
+ * @returns The system prompt string
+ */
+export const createModuleSystemPrompt = (
+  moduleContext: string,
+  searchContext: string = ""
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)